src/components/Header.tsx

```diff
@@ -19,11 +19,6 @@ const items = [
     label: "Projects",
     url: "/projects",
     enabled: true
-  },
-  {
-    label: "Mint",
```

Comment:
Do you think this should be kept in as an example of a disabled / potential tab @jakerockland ?

Reply:
Seems fine, generally a fan of more examples the better

Reply:
address with [59d2ffa](https://github.com/ArtBlocks/artblocks-engine-react/pull/10/commits/59d2ffa1d07b84eb86856625d2570d9849365f52)